### ISSUE TITLE
Class loading problem caused ozone ufs mount failure

### ISSUE THREAD

**Issue by Haoning-Sun** [Alluxio/alluxio#15960]
**Alluxio Version:**
master

**Describe the bug**
An exception occurred while mounting ozone ufs.
```
java.lang.IllegalArgumentException: Unable to create an UnderFileSystem instance for path: o3fs://jan.hive/
        at alluxio.underfs.UnderFileSystem$Factory.create(UnderFileSystem.java:120)
        at alluxio.underfs.AbstractUfsManager.getOrAdd(AbstractUfsManager.java:132)
        at alluxio.underfs.AbstractUfsManager.lambda$addMount$0(AbstractUfsManager.java:171)
        at alluxio.underfs.UfsManager$UfsClient.acquireUfsResource(UfsManager.java:61)
        at alluxio.master.file.DefaultFileSystemMaster.prepareForMount(DefaultFileSystemMaster.java:3022)
        at alluxio.master.file.DefaultFileSystemMaster.mountInternal(DefaultFileSystemMaster.java:3185)
        at alluxio.master.file.DefaultFileSystemMaster.mountInternal(DefaultFileSystemMaster.java:3147)
        at alluxio.master.file.DefaultFileSystemMaster.mount(DefaultFileSystemMaster.java:3122)
        at alluxio.master.file.FileSystemMasterClientServiceHandler.lambda$mount$11(FileSystemMasterClientServiceHandler.java:272)
        at alluxio.RpcUtils.callAndReturn(RpcUtils.java:123)
        at alluxio.RpcUtils.call(RpcUtils.java:84)
        at alluxio.RpcUtils.call(RpcUtils.java:59)
        at alluxio.master.file.FileSystemMasterClientServiceHandler.mount(FileSystemMasterClientServiceHandler.java:271)
        at alluxio.grpc.FileSystemMasterClientServiceGrpc$MethodHandlers.invoke(FileSystemMasterClientServiceGrpc.java:2408)
        at io.grpc.stub.ServerCalls$UnaryServerCallHandler$UnaryServerCallListener.onHalfClose(ServerCalls.java:182)
        at io.grpc.PartialForwardingServerCallListener.onHalfClose(PartialForwardingServerCallListener.java:35)
        at io.grpc.ForwardingServerCallListener.onHalfClose(ForwardingServerCallListener.java:23)
        at io.grpc.ForwardingServerCallListener$SimpleForwardingServerCallListener.onHalfClose(ForwardingServerCallListener.java:40)
        at alluxio.security.authentication.ClientIpAddressInjector$1.onHalfClose(ClientIpAddressInjector.java:57)
        at io.grpc.PartialForwardingServerCallListener.onHalfClose(PartialForwardingServerCallListener.java:35)
        at io.grpc.ForwardingServerCallListener.onHalfClose(ForwardingServerCallListener.java:23)
        at io.grpc.ForwardingServerCallListener$SimpleForwardingServerCallListener.onHalfClose(ForwardingServerCallListener.java:40)
        at alluxio.security.authentication.AuthenticatedUserInjector$1.onHalfClose(AuthenticatedUserInjector.java:67)
        at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.halfClosed(ServerCallImpl.java:331)
        at io.grpc.internal.ServerImpl$JumpToApplicationThreadServerStreamListener$1HalfClosed.runInContext(ServerImpl.java:797)
        at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
        at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:748)
        Suppressed: java.lang.ExceptionInInitializerError
                at java.lang.Class.forName0(Native Method)
                at java.lang.Class.forName(Class.java:264)
                at alluxio.underfs.hdfs.HdfsUnderFileSystem.<init>(HdfsUnderFileSystem.java:182)
                at alluxio.underfs.hdfs.HdfsUnderFileSystem.createInstance(HdfsUnderFileSystem.java:118)
                at alluxio.underfs.hdfs.HdfsUnderFileSystemFactory.create(HdfsUnderFileSystemFactory.java:42)
                at alluxio.underfs.UnderFileSystem$Factory.create(UnderFileSystem.java:105)
                ... 29 more
        Caused by: java.lang.RuntimeException: java.lang.ClassNotFoundException: Provider com.sun.ws.rs.ext.RuntimeDelegateImpl could not be instantiated: java.lang.ClassCastException: Cannot cast com.sun.jersey.core.impl.provider.header.LocaleProvider to com.sun.jersey.spi.HeaderDelegateProvider
                at javax.ws.rs.ext.RuntimeDelegate.findDelegate(RuntimeDelegate.java:122)
                at javax.ws.rs.ext.RuntimeDelegate.getInstance(RuntimeDelegate.java:91)
                at javax.ws.rs.core.MediaType.<clinit>(MediaType.java:44)
                ... 35 more
        Caused by: java.lang.ClassNotFoundException: Provider com.sun.ws.rs.ext.RuntimeDelegateImpl could not be instantiated: java.lang.ClassCastException: Cannot cast com.sun.jersey.core.impl.provider.header.LocaleProvider to com.sun.jersey.spi.HeaderDelegateProvider
                at javax.ws.rs.ext.FactoryFinder.newInstance(FactoryFinder.java:69)
                at javax.ws.rs.ext.FactoryFinder.find(FactoryFinder.java:155)
                at javax.ws.rs.ext.RuntimeDelegate.findDelegate(RuntimeDelegate.java:105)
                ... 37 more
        Caused by: java.lang.ClassCastException: Cannot cast com.sun.jersey.core.impl.provider.header.LocaleProvider to com.sun.jersey.spi.HeaderDelegateProvider
                at java.lang.Class.cast(Class.java:3369)
                at com.sun.jersey.spi.service.ServiceFinder$LazyObjectIterator.hasNext(ServiceFinder.java:833)
                at com.sun.jersey.core.spi.factory.AbstractRuntimeDelegate.<init>(AbstractRuntimeDelegate.java:76)
                at com.sun.ws.rs.ext.RuntimeDelegateImpl.<init>(RuntimeDelegateImpl.java:56)
                at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
                at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
                at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
                at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
                at java.lang.Class.newInstance(Class.java:442)
                at javax.ws.rs.ext.FactoryFinder.newInstance(FactoryFinder.java:65)
                ... 39 more
```

It is caused by #14950

**To Reproduce**
Steps to reproduce the behavior (as minimally and precisely as possible)

**Expected behavior**

**Urgency**
Describe the impact and urgency of the bug.

**Are you planning to fix it**
Please indicate if you are already working on a PR. 

**Additional context**

**Comment by HelloHorizon** [comment]
@Haoning-Sun do you plan to fix it?

**Comment by yuzhu** [comment]
closed via #16029 , please verify @Haoning-Sun
